docs/source/noaa_csp.rst

```diff
@@ -0,0 +1,202 @@
+created for a C48 atmosphere forecast-only configuration. Here the
+user must define the instance type (see the table above), the number
+of image disks and the image disk sizes.
+
+.. image:: _static/noaacsp_cluster_4.png
+
+Next the partitions for the cluster may be defined. A partition
+configuration for the aforementioned C48 atmosphere forecast-only
+application is illustrated in the figure above. Note that instance
+type beneath ``Controller Settings`` and ``Partitions`` must be
+identical. Other configurations are not supported by the
+global-workflow team. Once the partitions are configured, click the ``+
+Add Partition`` button in the upper-right corner.
+
+.. image:: _static/noaacsp_cluster_5.png
+
+Finally the storage for the global-workflow application. It is
+suggested that the ``Mount Point`` be ``/lustre``. Once the storage
+has been configured, click the ``+ Add Attached Storage`` button in
+the upper-right corner. The following illustrates a JSON version of
+the cluster configuration created from the steps above. When opening
+issues related to the NOAA CSP global-workflow applications please
+include the JSON content.
+
+.. image:: _static/noaacsp_cluster_6.png
+
+**************************
+Using the NOAA CSP Cluster
+**************************
+
+.. image:: _static/noaacsp_using_1.png
+
+To activate the cluster, click the button circled in
+:red-text:red. The cluster status is denoted by the color-coded button
+on the right. The amount of time required to start the cluster is
+variable and not immediate. For instances where NOAA CSP clusters do
+not initialize, useful output can be found beneath the ``Provision``
+tab beneath the ``Logs`` section as illustrated below. Once again,
+when opening issues related to the NOAA CSP cluster initialization
+please include this information.
+
+.. image:: _static/noaacsp_using_2.png
+
+***************************
+Running the Global Workflow
+***************************
+
+The global-workflow configuration currently requires that all initial
+conditions, observations, fixed-files, are staged in the appropriate
+paths prior to running the global-workflow. As suggested above, it is
+strongly recommended the the user configure their respective
+experiments to use the ``/lustre`` file system for the ``EXPDIR`` and
+``ROTDIR`` contents. The ``/contrib`` file system is suitable for
+compiling and linking the workflow components required of the
+global-workflow.
+
+The software stack supporting the ``develop`` branch of the
+global-workflow is provided for the user and is located beneath
+``/contrib/workflow/spack-stack``. The modules required for the
+global-workflow execution may be loaded as follows.
+
+.. code-block:: bash
+
+   user@host:$ module unuse /opt/cray/craype/default/modulefiles
+   user@host:$ module unuse /opt/cray/modulefiles
+   user@host:$ module use –a /contrib/global-workflow/spack-stack/miniconda/modulefiles/miniconda
```

Comment:
I do not have a `global-workflow` mounted in `/contrib`. I did find the module in `/contrib-epic/spack-stack/modulefiles/core/miniconda/` though.

Reply:
I think this is because you are using a different group than I (I am using `cpldcld`).

You could probably use the `miniconda` that you are point to, but it's not sustainable long term. 

My plan is to build against EPIC stack once everyone gets through the RTD draft.

Reply:
Nope, this is with `ca-ufs-cmd`